scripts/zkai_attention_d128_value_adapter_gate.py

```
@@ -0,0 +1,906 @@
+def build_adapter_analysis(
+    attention: dict[str, Any],
+    d128_input: dict[str, Any],
+    bridge: dict[str, Any],
+) -> dict[str, Any]:
+    _validate_expected(attention, EXPECTED_ATTENTION, "attention fixture")
+    _validate_expected(d128_input, EXPECTED_D128, "d128 input")
+    _validate_expected(bridge, EXPECTED_BRIDGE, "statement bridge")
+
+    bridge_summary = _dict(bridge.get("summary"), "bridge summary")
+    if bridge_summary.get("current_commitments_equal") is not False:
+        raise AttentionD128ValueAdapterError("statement bridge equality status drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)